tool/tsh/kube.go

```diff
@@ -406,6 +410,7 @@ func newKubeExecCommand(parent *kingpin.CmdClause) *kubeExecCommand {
 	c.Flag("tty", "Stdin is a TTY").Short('t').BoolVar(&c.tty)
 	c.Flag("reason", "The purpose of the session.").StringVar(&c.reason)
 	c.Flag("invite", "A comma separated list of people to mark as invited for the session.").StringVar(&c.invited)
+	c.Flag("participant-req", "Displays a verbose list of required participants in a moderated session.").BoolVar(&c.participantRequirements)
```

Comment:
Did you and @xinding33 agree on the naming of this `--participant-req` flag? I think we could choose a better name, but no need to bikeshed if it was already decided.

Reply:
@zmb3 Yep, we did agree on it.